js/models/wizard_topics.js

```diff
@@ -16,7 +16,7 @@ const allTopics = [
     tid: 't2', title: 'Travel records', journey: travelRecordsJourney,
   },
   {
-    tid: 't3', title: 'IRS records', journey: irsRecordsJourney,
+    tid: 't3', title: 'Tax records', journey: taxRecordsJourney,
```

Comment:
Here's where the intent model needs to match title: https://github.com/forumone/foia.gov/blob/d0269be897c41c5c20df79faa93a158863e18590/js/stores/wizard_store.js#L300

So, in case the intent model doesn't get updated, I suggest before this you do:
```js
// The intent model may return the old title. Update it as necessary.
flow = flow.replace(/IRS records/i, 'Tax records');
```